### PR TITLE
WS-021: Add confirm and delete buttons to sightings page, visible only to admins

### PR DESCRIPTION
**Pull request by AliThomson**
image
Hi, all functionality working as required. Buttons do need styling but I'll create a new trello card for that.
Thank you!

### CODE REVIEW

File: frontend/src/pages/Sightings/SightingListPage.tsx [hunk 1]
```diff
@@ -34,6 +55,33 @@ export function SightingListPage(): JSX.Element {
                 <p>
                   Seen by: {s.user.name} ({s.user.username})
                 </p>
+                {s.approvedBy !== null ? <p>Confirmed ☑</p> : <></>}
+
+                {isAdmin ? (
+                  <div>
+                    <button
+                      value={s.id}
```
Comment (AndrewYHuang) [inline frontend/src/pages/Sightings/SightingListPage.tsx:63]: We might not need `value={s.id}` anymore

File: frontend/src/pages/Sightings/SightingListPage.tsx [hunk 2]
```diff
@@ -34,6 +55,33 @@ export function SightingListPage(): JSX.Element {
...
+                      disabled={!!s.approvedBy}
+                      onClick={() => {
+                        confirmWhaleSighting(s.id);
+                      }}
+                      type="submit"
+                    >
+                      Confirm
+                    </button>
+                    <button
+                      value={s.id}
```
Comment (AndrewYHuang) [inline frontend/src/pages/Sightings/SightingListPage.tsx:73]: Ditto - Might not need this line

**Review by AndrewYHuang — CHANGES_REQUESTED** [review]

File: frontend/src/pages/Sightings/SightingListPage.tsx (hunk 2 above)
Comment (AliThomson) [reply frontend/src/pages/Sightings/SightingListPage.tsx:73]: As above

File: frontend/src/pages/Sightings/SightingListPage.tsx (hunk 1 above)
Comment (AliThomson) [reply frontend/src/pages/Sightings/SightingListPage.tsx:63]: You're right! Great, thank you - I've removed.

File: backend/Controllers/SightingsController.cs [hunk 3]
```diff
@@ -67,7 +67,16 @@ public ActionResult<List<ExtendedSightingResponse>> GetAllSightings()
                         Name = s.CreatedBy.Name,
                         Email = s.CreatedBy.Email,
                         Username = s.CreatedBy.Username
-                    }
+                 },
```
[the remark this answers is not in the document]
Comment (AliThomson) [reply backend/Controllers/SightingsController.cs:70]: Done

**Review by AndrewYHuang — APPROVED** [review]
Looks good, thanks! :)

**Pull request merged**